tests/test_pos_tagging_evaluator.py

```diff
@@ -0,0 +1,78 @@
+from itertools import chain
+from typing import  Dict, Union, List
+
+import pytest
+from embeddings.evaluator.sequence_tagging_evaluator import SequenceTaggingEvaluator
+from sklearn.metrics import classification_report, precision_recall_fscore_support
+
+
+@pytest.fixture  # type: ignore
+def data() -> Dict[str, List[List[str]]]:
+    return {
+        "y_true": [
```

Comment:
I really like the XD, but it is a little bit better to add real-life examples, because they helps with code readability :)